python/lsst/afw/image/mask.cc

```diff
@@ -61,16 +61,21 @@ void declareMask(py::module & mod, std::string const & suffix) {
             "fitsFile"_a, "metadata"_a=PTR(lsst::daf::base::PropertySet)(), "bbox"_a=lsst::afw::geom::Box2I(), "origin"_a=PARENT, "conformMasks"_a=false);
     cls.def(py::init<const Mask<MaskPixelT>&, const bool>(),
             "src"_a, "deep"_a=false);
+    cls.def(py::init<const Mask<MaskPixelT>&, const lsst::afw::geom::Box2I &, ImageOrigin const, const bool>(),
+            "src"_a, "bbox"_a, "origin"_a=PARENT, "deep"_a=false);
     cls.def(py::init<ndarray::Array<MaskPixelT,2,1> const &, bool, lsst::afw::geom::Point2I const &>(),
             "array"_a, "deep"_a=false, "xy0"_a=lsst::afw::geom::Point2I());
 
     /* Operators */
-    cls.def("__ior__", (Mask<MaskPixelT>& (Mask<MaskPixelT>::*)(Mask<MaskPixelT> const &)) &Mask<MaskPixelT>::operator|=);
-    cls.def("__ior__", (Mask<MaskPixelT>& (Mask<MaskPixelT>::*)(MaskPixelT const)) &Mask<MaskPixelT>::operator|=);
-    cls.def("__iand__", (Mask<MaskPixelT>& (Mask<MaskPixelT>::*)(Mask<MaskPixelT> const &)) &Mask<MaskPixelT>::operator&=);
-    cls.def("__iand__", (Mask<MaskPixelT>& (Mask<MaskPixelT>::*)(MaskPixelT const)) &Mask<MaskPixelT>::operator&=);
-    cls.def("__ixor__", (Mask<MaskPixelT>& (Mask<MaskPixelT>::*)(Mask<MaskPixelT> const &)) &Mask<MaskPixelT>::operator^=);
-    cls.def("__ixor__", (Mask<MaskPixelT>& (Mask<MaskPixelT>::*)(MaskPixelT const)) &Mask<MaskPixelT>::operator^=);
+    cls.def("__ior__", [](Mask<MaskPixelT> & self, Mask<MaskPixelT> & other) { return self |= other; }, py::is_operator());
+    cls.def("__ior__", [](Mask<MaskPixelT> & self, MaskPixelT const other) { return self |= other; }, py::is_operator());
+    cls.def("__ior__", [](Mask<MaskPixelT> & self, int other) { return self |= other; }, py::is_operator());
+    cls.def("__iand__", [](Mask<MaskPixelT> & self, Mask<MaskPixelT> & other) { return self &= other; }, py::is_operator());
+    cls.def("__iand__", [](Mask<MaskPixelT> & self, MaskPixelT const other) { return self &= other; }, py::is_operator());
+    cls.def("__iand__", [](Mask<MaskPixelT> & self, int other) { return self &= other; }, py::is_operator());
+    cls.def("__ixor__", [](Mask<MaskPixelT> & self, Mask<MaskPixelT> & other) { return self ^= other; }, py::is_operator());
+    cls.def("__ixor__", [](Mask<MaskPixelT> & self, MaskPixelT const other) { return self ^= other; }, py::is_operator());
+    cls.def("__ixor__", [](Mask<MaskPixelT> & self, int other) { return self ^= other; }, py::is_operator());
```

Comment:
clang-format! :) I think I'll just run that anyway. The small differences with our style guide won't pop up in the pybind11 code so it shouldn't be a problem.